help/sources/tutorials/ui/create/crm/sugarcrm-accounts-contacts.md

```diff
@@ -0,0 +1,109 @@
+
+![Platform UI screenshot for Connect SugarCRM Accounts & Contacts account with a new account](../../../../images/tutorials/create/sugarcrm-accounts-contacts/new.png?lang=en)
+
+Finally, depending on your dataset you must choose as a mandatory input between the object types below:
+| object_type | Description |
+| --- | --- |
+| `Accounts` | Companies with whom your organization has a relationship. |
+| `Contacts` | Individual people with whom your organization has an established relationship. |
+
+Platform UI screenshot showing configuration when using the *Account* object type.
+![Platform UI screenshot for SugarCRM Accounts & Contacts showing configuration with Account option selected](../../../../images/tutorials/create/sugarcrm-accounts-contacts/configuration-accounts.png?lang=en)
+
+Platform UI screenshot showing configuration when using the *Contact* object type.
+![Platform UI screenshot for SugarCRM Accounts & Contacts showing configuration with Contacts option selected](../../../../images/tutorials/create/sugarcrm-accounts-contacts/configuration-contacts.png?lang=en)
+
+## Next steps
+
+By following this tutorial, you have established a connection to your [!DNL SugarCRM Accounts & Contacts] account. You can now continue on to the next tutorial and [configure a dataflow to bring data into Platform](/help/sources/ui-tutorials/dataflow/crm.md).
+
+## Additional resources
+
+### Guardrails {#guardrails}
+
+The [!DNL SugarCRM] API throttle rates are 90 calls per minute or 2000 calls per day, whichever happens first. However, this restriction has been circumvented by adding a parameter into the connection spec that will delay request time so that the rate limit is never reached.
+
+### Validation {#validation}
+
+To validate that you have correctly set up the source and [!DNL SugarCRM Accounts & Contacts] data is being ingested, follow the steps below:
+
+1. In the Platform UI, select View Dataflows besides the [!DNL SugarCRM Accounts & Contacts] card menu on the Catalog page. You can then select [!UIControl Preview dataset] to verify the data that was ingested.
+
+1. Depending on the object type you are working with you can verify the aggregated data against the counts visible on the [!DNL SugarMarket] Accounts page below : 
+   ![Screenshot from the SugarMarket Accounts page displaying list of accounts](../../../../images/tutorials/create/sugarcrm-accounts-contacts/sugarmarket-accounts.png?lang=en)
+
+   Or from the Contacts page below :
+   ![Screenshot from the SugarMarket Contacts page displaying list of contacts](../../../../images/tutorials/create/sugarcrm-accounts-contacts/sugarmarket-contacts.png?lang=en)
+
+>[!NOTE] 
+> The [!DNL SugarMarket] pages do not include the deleted object counts. However, data retrieved through this source will also include the deleted count, these would be marked with a deleted flag.
```

Comment:
Need to add another ">" in between here, and also need to remove the space between ">" and the text, so:

>[!NOTE]
>
>The [!DNL SugarMarket]...